strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/testing/TestCaseWithMavenArtifactGenerationAndIndexing.java

```diff
@@ -187,12 +187,12 @@ protected MutableRepository createRepository(String storageId,
         return repository;
     }
 
-    protected MutableRepository createGroup(String storageId,
+    protected RepositoryDto createGroup(String storageId,
                                             String repositoryId,
                                             String... leafs)
```

Comment:
Cosmetic: Could you please re-indent the method parameters? Thanks! :)

Reply:
fixed